website/content/guides/1.7-template-upgrade.mdx

```diff
@@ -8,50 +8,71 @@ sidebar_title: Upgrade Your Template to use Packer init
+[Packer Plugin SDK](https://github.com/hashicorp/packer-plugin-sdk), and therefore are compatible with Packer's API
+version v5.0. The plugin repository on GitHub also needs to use a specific release format. If you are not sure whether
+the plugin you use fits those requirements, you can reach out to your maintainer to ask. You can also look for clues
+that the plugin is ready for use with `packer init`:
 
-#### The plugin I need is not upgraded to API v5.0
+* Check the plugin's CHANGELOG for notes about migrating to the packer-plugin-sdk.
+* Check the name of the repository the plugin resides in. Chances are that if the repository follows the naming
+  convention `packer-plugin-*` (e.g. `packer-plugin-comment`), then the plugin has been upgraded to be compatible with
+  `packer init`. If the repository has a name that references a specific Packer component (for example,
+  `packer-provisioner-*`, `packer-builder-*`, or `packer-post-processor-*`) then the plugin likely still needs to be
+  upgraded to be compatible with `packer init`. Reach out to your plugin maintainer to request that they upgrade; the
+  Packer team has written a maintainer-focused guide [here](/guides/1.7-plugin-upgrade).
 
-With the SDK released at the same time as the `init` command, plugin users may face a gap between the release and
-plugins upgrade. Maintainers will have to upgrade their plugins, and the process is a little more complicated than this one.
-The Packer team is getting in touch with all currently known plugin maintainers to provide support during their upgrade process.
+If the plugin(s) have been upgraded, then they can be used with the `required_plugins` block in your upgraded template.
+
+#### What if the plugin you need is not upgraded to API v5.0?
+
+Since the SDK is being released at the same time as the `init` command, plugin users may face a gap between when the
+Packer core v1.7.0 is released, and the plugins you use are upgraded by their maintainers. The Packer team is getting
+in touch with all currently known plugin maintainers to provide support during their upgrade process.
 
 If you are willing to upgrade your template but found out that the plugin you are using hasn't been upgraded yet,
-we suggest you reach out to the plugin maintainers and ask for an upgrade.
+we suggest you reach out to the plugin maintainers and ask for an upgrade; the Packer team has written a
+maintainer-focused guide [here](/guides/1.7-plugin-upgrade).
 
-Check the table below to better understand what behavior to expect from a Packer version with the current state
-of your template and the plugins you are using.
+Check the table below to better understand whether your plugin is compatible with `packer init`, with manual
+installation, or with both for the the Packer version you are using.
 
-| Single Component Plugin - API v4 | Single Component Plugin - API v5.0     | Multi Component Plugin - API v5.0      | Packer Version   |
```

Comment:
I moved the "version" to the left side since normally in tables that's where I would expect the row headers to go. I also made the wording a little more verbose because when I first read over this I got confused about what compatibility we were actually talking about.

Reply:
That's what I would have done also !

Reply:
Nice! Thanks.